src/main/kotlin/org/taktik/icure/asynclogic/samv2/impl/SamV2LogicImpl.kt

```diff
@@ -160,8 +166,17 @@ class SamV2LogicImpl(
 		return vmpDAO.findVmpsByGroupCode(vmpgCode, paginationOffset)
 	}
 
-	override fun findAmpsByLabel(language: String?, label: String?, paginationOffset: PaginationOffset<List<String>>): Flow<ViewQueryResultEvent> {
-		return ampDAO.findAmpsByLabel(language, label, paginationOffset)
+	override fun findAmpsByLabel(language: String?, label: String, paginationOffset: PaginationOffset<List<String>>): Flow<Amp> = flow {
```

Comment:
Cover it as well with a test to make sure we're retrieving all corresponding Amps